News/InterestsPage.xaml

```diff
@@ -0,0 +1,64 @@
+﻿<Page
+    x:Class="News.InterestsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:News"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="300"/>
+            <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="165"/>
+            <RowDefinition/>
+        </Grid.RowDefinitions>
+
+        <StackPanel Orientation="Vertical" 
+                    Grid.Row="0"
+                    Grid.Column="0" 
+                    Grid.ColumnSpan="2">
+            <TextBlock Text="Мои интересы"
+                        Foreground="DarkSlateGray" 
+                        FontWeight="Bold" 
+                        Padding="20 10 0 0"
```

Comment:
For on-screen dimensions, use values that are multiples of 4 (like "20, 12, 0, 0")